src/components/renderer/form-button.vue

```diff
@@ -9,40 +9,25 @@
 <script>
 import Mustache from 'mustache';
 import { getValidPath } from '@/mixins';
-import { isProxy } from 'is-proxy';
 
 export default {
   mixins: [getValidPath],
   props: ['variant', 'label', 'event', 'eventData', 'name', 'fieldValue', 'value', 'tooltip', 'transientData'],
   watch: {
-    '$attrs.validate': {
+    "$store.state.globalErrorsModule": {
```

Comment:
So. We have here multiple unnecessary watchers. One, which is watching the main default which is `globalErrorsModule`, the second one which is watching the state `valid`, and the rest which are all the properties and state. Party of watchers 🥳 
This is a no-go.

I would create the following.
```javascript
computed: mapState('globalErrorsModule', ['valid'])
```
then in the watchers piece 
```javascript
watch: {
  "valid": {
      this.isInvalid = !valid;
    }
}
```

Reply:
Thanks! fixed